src/javascripts/components/Chat.js

```diff
@@ -1,19 +1,105 @@
-var React = require('react');
-var ChatBox_settings = require('./ChatBox_settings');
-var ChatBox = require('./ChatBox');
-var ChatInput = require('./ChatInput');
-
+import React from 'react';
+import ChatBoxSettings from './ChatBoxSettings';
+import ChatBox from './ChatBox';
+import ChatInput from './ChatInput';
 
 class Chat extends React.Component {
+  constructor() {
+      super()
+      this.state = {
+        open: true,
+        messages: [
+          {
+            user: "blue",
+            message: "blah blah blah"
+          }, 
+          {
+            user: "yellow",
+            message: "Lorem ipsum dipsum"
+          },
+          {
+            user: "yellow",
+            message: "Hallo lorem"
+          }
+        ]
+      }
+  }
+
+  addMessages = (text) => {
+    const newMsg = {
+      user: "blue",
+      message: text
+    }
+    const resMsgs = [
+    {
+      user: "yellow",
+      message: "Waddup"
+    },
+    {
+      user: "yellow",
+      message: "Insert meme here"
+    },
+    {
+      user: "yellow",
+      message: "You're so funny!"
+    },
+    {
+      user: "yellow",
+      message: "So... the weather today?"
+    },
+    {
+      user: "yellow",
+      message: "Ding Dong Diddly Doo I'm rhyming words and singing to you."
+    },
+    {
+      user: "yellow",
+      message: "Mathmatical!"
+    },
+    {
+      user: "yellow",
+      message: "How are you today?"
+    },
+    {
+      user: "yellow",
+      message: "Hahahaha"
+    },
+    {
+      user: "yellow",
+      message: "I pronounce 'memes' like 'me' 'mees'"
+    },
+    {
+      user: "yellow",
+      message: "That makes a lot of sense!"
+    },
+    {
+      user: "yellow",
+      message: "Diddly doo, this rhymes with poo and you."
+    },
+    {
+      user: "yellow",
+      message: "Rocket Racoon rams rockets rinto rhis rpockets"
+    },
```

Comment:
indent the objects in this array a bit.

This whole list is also just a constant that is never manipulated or changed, so it would be good to move it somewhere else that is more out-of-the-way.  It adds a lot of lines of code before we get to more logic in this file.

I'd move it to a new file, something like: `constants/replies.js`.  This does add a new `constants` directory, which can be a sibling to `actions`, `presentors`, etc.